### ISSUE TITLE
Error while build android@7.1.1 

### ISSUE THREAD

**Issue by bhumin3i** [transistorsoft/cordova-background-geolocation-lt#930]
**### ERROR :** 
> :app:processDebugResources/Users/rac/.gradle/caches/transforms-1/files-1.1/appcompat-v7-28.0.0.aar/fd44eaxyz§/res/values-v28/values-v28.xml:9:5-12:13: AAPT: error: resource android:attr/dialogCornerRadius not found.

>  
> /Users/rac/Desktop/git/xxxx-frontend/platforms/android/app/build/intermediates/incremental/mergeDebugResources/merged.dir/values-v28/values-v28.xml:11: AAPT: error: resource android:attr/dialogCornerRadius not found.
>     
> /Users/rac/.gradle/caches/transforms-1/files-1.1/appcompat-v7-28.0.0.aar/fd44e49zyzx/res/values/values.xml:1304:5-69: AAPT: error: resource android:attr/fontVariationSettings not found.
>     

**project.properties file**
```
android.library.reference.1=CordovaLib
android.library.reference.2=app
cordova.gradle.include.1=cordova-plugin-background-fetch/ABC-build.gradle
cordova.system.library.1=com.google.android.gms:play-services-location:11.8.0
cordova.system.library.2=com.android.support:appcompat-v7:27.+
cordova.system.library.3=com.squareup.okhttp3:okhttp:3.12.+
cordova.system.library.4=org.greenrobot:eventbus:3.0.0
cordova.system.library.5=com.intentfilter:android-permissions:1.0.43
cordova.system.library.6=org.slf4j:slf4j-api:1.7.25
cordova.system.library.7=com.github.tony19:logback-android:1.3.0-2
cordova.gradle.include.2=cordova-background-geolocation-lt/ABC-build.gradle
cordova.system.library.8=com.android.support:support-annotations:27.+
cordova.gradle.include.3=cordova-plugin-ms-adal/ABC-adal.gradle
cordova.gradle.include.4=cordova-support-google-services/ABC-build.gradle
cordova.gradle.include.5=phonegap-plugin-multidex/ABC-multidex.gradle
cordova.system.library.9=com.android.support:support-v13:27.+
cordova.system.library.10=me.leolin:ShortcutBadger:1.1.17@aar
cordova.system.library.11=com.google.firebase:firebase-messaging:11.8.0
cordova.system.library.12=io.sentry:sentry-android:1+
```

**Comment by christocracy** [comment]
```diff
+cordova.system.library.1=com.google.android.gms:play-services-location:16.0.0
+cordova.system.library.11=com.google.firebase:firebase-messaging:18.0.0
```

**Comment by christocracy** [comment]
And **DO NOT IGNORE THE ISSUE TEMPLATE!**.

* Plugin version:
* Platform: iOS or Android
* OS version:
* Device manufacturer / model:
* Cordova version (`cordova -v`):
* Cordova platform version (`cordova platform ls`):
* Plugin config

**Comment by christocracy** [comment]
```
# Project target.
+target=android-28
```

**Comment by christocracy** [comment]
If you upgrade to latest Cordova 9 (`cordova-android@8.0`), it defaults to `target=android-28`.

(always be sure to remove / re-add platforms when upgrading cordova).

```
$ cordova -v
9.0.0 (cordova-lib@9.0.1)

$ cordova platforms
Installed platforms:
  android 8.0.0

```

**Comment by christocracy** [comment]
I've [reverted to previous version](https://github.com/transistorsoft/cordova-background-geolocation-lt/blob/master/CHANGELOG.md#304---2019-05-10) of the plugin's `android-permissions` library, which was requiring `targetSdkVersion 28`.

Installing `3.0.4` should fix you.

**Comment by bhandaribhumin** [comment]
@christocracy  Thanks for fix but when I was use 3.0.4 still getting same error

I'm using android@7.1.1

$cordova -v
9.0.0 (cordova-lib@9.0.1)

$ cordova platforms
Installed platforms:
  android 7.1.1
  ios 4.5.5
Available platforms: 
  browser ^6.0.0
  electron ^1.0.0
  osx ^5.0.0
  windows ^7.0.0

.properties file
target=android-27
android.library.reference.1=CordovaLib
android.library.reference.2=app
cordova.gradle.include.1=cordova-plugin-background-fetch/xxxx-build.gradle
cordova.system.library.1=com.google.android.gms:play-services-location:16.0.0
cordova.system.library.2=com.android.support:appcompat-v7:27.+
cordova.system.library.3=com.squareup.okhttp3:okhttp:3.12.+
cordova.system.library.4=org.greenrobot:eventbus:3.0.0
cordova.system.library.5=com.intentfilter:android-permissions:0.1.8
cordova.system.library.6=org.slf4j:slf4j-api:1.7.25
cordova.system.library.7=com.github.tony19:logback-android:1.3.0-2
cordova.gradle.include.2=cordova-background-geolocation-lt/xxxx-build.gradle
cordova.system.library.8=com.android.support:support-annotations:27.+
cordova.gradle.include.3=cordova-plugin-ms-adal/xxxx-adal.gradle
cordova.gradle.include.4=cordova-support-google-services/xxxx-build.gradle
cordova.gradle.include.5=phonegap-plugin-multidex/xxxx-multidex.gradle
cordova.system.library.9=com.android.support:support-v13:27.+
cordova.system.library.10=me.leolin:ShortcutBadger:1.1.17@aar
cordova.system.library.11=com.google.firebase:firebase-messaging:18.0.0
cordova.system.library.12=io.sentry:sentry-android:1+

**Comment by bhumin3i** [comment]
@christocracy  Thanks for fix but when I was use 3.0.4 still getting error this time it's different I think this is from push :

:app:compileDebugJavaWithJavac/Users/rac/Desktop/git/my-app-frontend/platforms/android/app/src/main/java/com/adobe/phonegap/push/PushInstanceIDListenerService.java:9: error: cannot find symbol
import com.google.firebase.iid.FirebaseInstanceIdService;
                              ^
  symbol:   class FirebaseInstanceIdService
  location: package com.google.firebase.iid
/Users/rac/Desktop/git/my-app-frontend/platforms/android/app/src/main/java/com/adobe/phonegap/push/PushInstanceIDListenerService.java:15: error: cannot find symbol

I'm using android@7.1.1

$cordova -v
9.0.0 (cordova-lib@9.0.1)

$ cordova platforms
Installed platforms:
  android 7.1.1
  ios 4.5.5
Available platforms: 
  browser ^6.0.0
  electron ^1.0.0
  osx ^5.0.0
  windows ^7.0.0

.properties file
target=android-27
android.library.reference.1=CordovaLib
android.library.reference.2=app
cordova.gradle.include.1=cordova-plugin-background-fetch/xxxx-build.gradle
cordova.system.library.1=com.google.android.gms:play-services-location:16.0.0
cordova.system.library.2=com.android.support:appcompat-v7:27.+
cordova.system.library.3=com.squareup.okhttp3:okhttp:3.12.+
cordova.system.library.4=org.greenrobot:eventbus:3.0.0
cordova.system.library.5=com.intentfilter:android-permissions:0.1.8
cordova.system.library.6=org.slf4j:slf4j-api:1.7.25
cordova.system.library.7=com.github.tony19:logback-android:1.3.0-2
cordova.gradle.include.2=cordova-background-geolocation-lt/xxxx-build.gradle
cordova.system.library.8=com.android.support:support-annotations:27.+
cordova.gradle.include.3=cordova-plugin-ms-adal/xxxx-adal.gradle
cordova.gradle.include.4=cordova-support-google-services/xxxx-build.gradle
cordova.gradle.include.5=phonegap-plugin-multidex/xxxx-multidex.gradle
cordova.system.library.9=com.android.support:support-v13:27.+
cordova.system.library.10=me.leolin:ShortcutBadger:1.1.17@aar
cordova.system.library.11=com.google.firebase:firebase-messaging:18.0.0
cordova.system.library.12=io.sentry:sentry-android:1+

ANDROID SDK:

![Screen Shot 2019-05-15 at 4 26 04 PM](https://user-images.githubusercontent.com/3336047/57770570-3ef5ae00-772e-11e9-8934-8dbaccf89bb9.png)

**Comment by bhandaribhumin** [comment]
@christocracy Thanks for help.
after change   - 'com.google.firebase:firebase-messaging:18.0.0' => 15.0.0 it works :)
